WreckEngine/External.h

```diff
@@ -6,41 +6,50 @@
 #include "Random.h"
 
 struct Window {
-	static GLFWwindow* window;
-	static int width, height;
-	static float aspect;
+    static GLFWwindow* window;
+    static int width, height;
+    static int frameWidth, frameHeight;
+    static float aspect;
+    static int cursorMode;
```

Comment:
Ehh, maybe? I put it in here because GLFW associates cursor mode with a window (because it's an input mode).